### PR TITLE
AD-49:Activation/Deactivation features imported from previous plugin

### PR DESCRIPTION
**Pull request by pranayhbwsl**

### CODE REVIEW

**Review by ratnesh-kadam — APPROVED** [review]

**Review by ratnesh-kadam — APPROVED** [review]

**Review by wpekaclub — DISMISSED** [review]

File: admin/class-wpadcenter-admin.php [hunk 1]
```diff
@@ -25,27 +25,28 @@ class Wpadcenter_Admin {
 	/**
 	 * The ID of this plugin.
 	 *
-	 * @since    1.0.0
-	 * @access   private
-	 * @var      string    $plugin_name    The ID of this plugin.
+	 * @since  1.0.0
+	 * @access private
+	 * @var    string    $plugin_name    The ID of this plugin.
```
Comment (sandeeprhbwsl) [inline admin/class-wpadcenter-admin.php:30]: Plugin name is incorrect

**Review by sandeeprhbwsl — COMMENTED** [review]
Analytics part of the plugin - code should be a submodule. It's a common project, shouldn't be a part of this source code.

**Review by sandeeprhbwsl — COMMENTED** [review]
For the build process, the css & js should be minified. The enqueue for production should consider the minified files.

**Review by sandeeprhbwsl — APPROVED** [review]
Reviewed the changes. Additional PR needed for some of the changes.

**Pull request merged**